libs/vkd3d/resource.c

```diff
@@ -4144,20 +4193,44 @@ HRESULT vkd3d_init_null_resources(struct vkd3d_null_resources *null_resources,
     if (!use_sparse_resources && FAILED(hr = vkd3d_allocate_image_memory(device, null_resources->vk_2d_storage_image,
             &heap_properties, D3D12_HEAP_FLAG_NONE, &null_resources->vk_2d_storage_image_memory, NULL, NULL)))
         goto fail;
+    if (!vkd3d_create_vk_image_view(device, null_resources->vk_2d_storage_image,
```

Comment:
Are these resources being used anywhere? I seem them being created and destroyed.

Reply:
Not yet, but I'm building the root signature rework around this currently, in order to use descriptor update templates as mentioned in the comment.

Of course we can treat this as an RFC for now and merge it later.

Reply:
Alright, just wanted to make sure. No reason not to merge it.